[on hold] check --version flag

ltalirz opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

ltalirz committed:
check --version flag

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ltalirz committed:
switch to gcc for c as well

see [1] and [2]

[1] https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html#using-your-customized-compiler-package-with-conda-build-3
[2] https://github.com/conda-forge/compilers-feedstock/blob/86145563bb81f82a29dab05810326ae68c907cb0/recipe/meta.yaml#L5-L24

ltalirz commented:
@conda-forge-admin , please rerender

Commit:
MNT: Re-rendered with conda-build 3.21.4, conda-smithy 3.8.6, and conda-forge-pinning 2021.02.05.12.35.12

ltalirz commented:
Seems [there is no gcc for osx64](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=274217&view=logs&j=58ac6aab-c4bc-5de2-2894-98e408cc8ec9&t=933f325c-924e-533d-4d95-e93b5843ce8b&l=973) :-(

I.e. we'll either need to live with the fact that `--version` is broken, or wait until gcc becomes available.
Or figure out how to fix it with clang.
